src/main/kotlin/blackjack/model/Stats.kt

```diff
@@ -0,0 +1,45 @@
+package blackjack.model
+
+class Stats(val players: List<Player>, val dealer: Dealer) {
+    val playerBoard: Map<Player, Result> get() = calculatePlayerBoard()
```

Comment:
Is playerBoard really necessary?
Try to minimize internal state as much as possible. 🙂